src/walletSdk/Recovery/AccountRecover.ts

```diff
@@ -0,0 +1,248 @@
+import { AxiosInstance } from "axios";
+import { ServerApi, Transaction } from "stellar-sdk";
+import { AccountRecordSigners } from "stellar-sdk/lib/types/account";
+
+import {
+  RecoveryServer,
+  RecoveryServerKey,
+  RecoveryServerMap,
+  RecoveryServerSigning,
+  RecoveryServerSigningMap,
+} from "walletSdk/Types";
+import {
+  LostSignerKeyNotFound,
+  NoDeviceKeyForAccountError,
+  NotAllSignaturesFetchedError,
+  RecoveryServerNotFoundError,
+  ServerRequestFailedError,
+  UnableToDeduceKeyError,
+} from "../Exceptions";
+import {
+  AccountKeypair,
+  PublicKeypair,
+  SponsoringBuilder,
+  Stellar,
+} from "../Horizon";
+
+export class AccountRecover {
```

Comment:
@Ifropc what was the reasoning for separating `AccountRecovery` from `Recovery` classes? Is it just for readability?

Should `AccountRecovery` be an abstract class?

Reply:
Yes just for readability

Reply:
@CassioMG can you add `abstract` then

Reply:
Nice catch, done!